recipes/y-py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "y-py" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yjs/y-crdt/archive/v{{ version }}.tar.gz
```

Comment:
it would be _lovely_ if the upstream on pypi were used, just so we have some chain for the automation to follow. tags are great... but can be moved...

Reply:
... of course, it would need the source _anyway_ as the pre-compiled wheel isn't going to do much, but it does demonstrate that it is "well-packaged" upstream, has the licenses in the canonical distribution, etc.

Reply:
I was thinking about that too, but the PyPI package is a bit behind: https://pypi.org/project/y-py/#files.